package.json

```
@@ -9,12 +9,14 @@
     "dist"
   ],
   "scripts": {
+    "dev": "vite",
```

If we want, we can have this automatically open in the browser:

```suggestion
    "dev": "vite --open",
```